docker-compose.yml

```diff
@@ -1,21 +0,0 @@
-version: '3'
-services:
-  piwik:
-    image: udata/piwik-test
```

Comment:
Don't we want to keep a piwik/matomo image for testing purposes?

Reply:
I opted for total removal because this image is quite a complex hack that was done specifically to debug metric computations (I should know, I did it ;-)). I don't think it's necessary anymore but YMMV.